corehq/apps/adm/README.md

```diff
@@ -1,17 +0,0 @@
-# Active Data Management Reports
-
-## Enabling Projects
-
-You can enable projects to view ADM reports in the Reports section of HQ
-by adding the following document to your couchdb:
-
-```{
-    "_id": "ADM_ENABLED_DOMAINS",
-    "domains": []
-}```
```

Comment:
I ran

```bash
zgrep /adm/ cchq-apache.log.[1-9] | grep -v 'GET /tour/adm/' | grep -v 'GET /static/' | grep crs | grep -o '\[.*\]' | sort
```
and looks like crs-remind `/adm/` urls haven't been accessed period since 2013.